usaspending_api/recipient/delta_models/sam_recipient.py

```diff
@@ -1,29 +1,90 @@
 SAM_RECIPIENT_COLUMNS = {
-    "awardee_or_recipient_uniqu": "STRING",
-    "legal_business_name": "STRING",
-    "ultimate_parent_unique_ide": "STRING",
-    "ultimate_parent_legal_enti": "STRING",
-    "broker_duns_id": "INTEGER NOT NULL",
-    "update_date": "DATE NOT NULL",
-    "address_line_1": "STRING",
-    "address_line_2": "STRING",
-    "city": "STRING",
-    "congressional_district": "STRING",
-    "country_code": "STRING",
-    "state": "STRING",
-    "zip": "STRING",
-    "zip4": "STRING",
-    "business_types_codes": "ARRAY<STRING>",
-    "dba_name": "STRING",
-    "entity_structure": "STRING",
-    "uei": "STRING",
-    "ultimate_parent_uei": "STRING",
+    "awardee_or_recipient_uniqu": {"delta": "STRING", "postgres": "TEXT"},
+    "legal_business_name": {"delta": "STRING", "postgres": "TEXT"},
+    "ultimate_parent_unique_ide": {"delta": "STRING", "postgres": "TEXT"},
+    "broker_duns_id": {"delta": "STRING", "postgres": "TEXT"},
+    "update_date": {"delta": "STRING", "postgres": "TEXT"},
+    "address_line_1": {"delta": "STRING", "postgres": "TEXT"},
+    "address_line_2": {"delta": "STRING", "postgres": "TEXT"},
+    "city": {"delta": "STRING", "postgres": "TEXT"},
+    "congressional_district": {"delta": "STRING", "postgres": "TEXT"},
+    "country_code": {"delta": "STRING", "postgres": "TEXT"},
+    "state": {"delta": "STRING", "postgres": "TEXT"},
+    "zip": {"delta": "STRING", "postgres": "TEXT"},
+    "zip4": {"delta": "STRING", "postgres": "TEXT"},
+    "business_types_codes": {"delta": "ARRAY<STRING>", "postgres": "TEXT[]"},
+    "dba_name": {"delta": "STRING", "postgres": "TEXT"},
+    "entity_structure": {"delta": "STRING", "postgres": "TEXT"},
+    "uei": {"delta": "STRING", "postgres": "TEXT"},
+    "ultimate_parent_uei": {"delta": "STRING", "postgres": "TEXT"},
 }
 
-sam_recipient_sql_string = rf"""
+SAM_RECIPIENT_DELTA_COLUMNS = {k: v["delta"] for k, v in SAM_RECIPIENT_COLUMNS.items()}
+SAM_RECIPIENT_POSTGRES_COLUMNS = {k: v["postgres"] for k, v in SAM_RECIPIENT_COLUMNS.items()}
+
+sam_recipient_create_sql_string = rf"""
     CREATE OR REPLACE TABLE {{DESTINATION_TABLE}} (
-        {", ".join([f'{key} {val}' for key, val in SAM_RECIPIENT_COLUMNS.items()])}
+        {", ".join([f'{key} {val}' for key, val in SAM_RECIPIENT_DELTA_COLUMNS.items()])}
     )
     USING DELTA
     LOCATION 's3a://{{SPARK_S3_BUCKET}}/{{DELTA_LAKE_S3_PATH}}/{{DESTINATION_DATABASE}}/{{DESTINATION_TABLE}}'
     """
+sam_recipient_load_sql_string = [
+    rf"""
+CREATE OR REPLACE TEMPORARY VIEW Source_data AS (
```

Comment:
Would lowercase this name from `Source_data` to `source_data`

Reply:
renamed view to source_data.